enrichments/trace/internal/elastic/attributes.go

```diff
@@ -47,4 +47,15 @@ const (
 	AttributeSpanDestinationServiceResource = "span.destination.service.resource"
 	AttributeSpanDurationUs                 = "span.duration.us"
 	AttributeSpanRepresentativeCount        = "span.representative_count"
+
+	// span event attributes
+	AttributeParentID              = "parent.id"
```

Comment:
I wonder if we should solve that by adding a `parent.id` -> `span_id` alias in the logs data stream.

Reply:
> I'm afraid the "parent" in parent.id is not the span, but the parent of span, i.e. parent.id != span_id.

I don't think this is true for errors. In case of errors, we specifically override the parent ID of the `SpanEvent` to be the `SpanID` (or `TransactionID for elastic transaction): https://github.com/elastic/apm-data/blob/bd5e34116889942c53b186deab0f76beaf182de0/input/otlp/traces.go#L1255-L1259

Reply:
oh wow, I missed that. Then is it a bug for logs from span events?

Reply:
does this fix in apm-data look right? https://github.com/elastic/apm-data/pull/374
If so, yes, I think we can have `parent.id` alias -> `span.id`.

Reply:
we are discussing on slack the alternative to make the UI use `span.id` directly: https://elastic.slack.com/archives/C95SB62AG/p1727693736192519?thread_ts=1727462154.365409&cid=C95SB62AG